fiftyone/utils/cvat.py

```
@@ -7268,6 +7279,14 @@ def _to_pairs_of_points(self, points):
         reshaped_points = np.reshape(points, (-1, 2))
         return reshaped_points.tolist()
 
+    def _skeleton_points(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_skeleton_points`

(ANN202)